OpenRA.Mods.Common/Widgets/Logic/Ingame/AddFactionSuffixLogic.cs

```diff
@@ -29,13 +29,22 @@ public AddFactionSuffixLogic(Widget widget, World world)
 				((ImageWidget)widget).ImageCollection += suffix;
 			else if (widget is BackgroundWidget)
 				((BackgroundWidget)widget).Background += suffix;
+			else if (widget is TextFieldWidget)
+				((TextFieldWidget)widget).Background += suffix;
+			else if (widget is ScrollPanelWidget)
```

Comment:
Can we please switch these all over to the newer pattern matching syntax while we're here?
```csharp
else if (widget is ScrollPanelWidget spw)
{
	spw.Button += suffix;
	spw.Background += suffix;
	spw.ScrollBarBackground += suffix;
	spw.Decorations += suffix;
}
```